.github/ci/files/var/classes/DataObject/OnlineShopOrder/PaymentProvider.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Pimcore\Model\DataObject\OnlineShopOrder;
+
+use Pimcore\Model\DataObject\Exception\InheritanceParentNotFoundException;
+
+class PaymentProvider extends \Pimcore\Model\DataObject\Objectbrick {
+
+protected $brickGetters = ['PaymentProviderPayPalSmartButton'];
```

Comment:
```suggestion
protected $brickGetters = ['PaymentProviderMpay24'];
```
Please change at other places as well